Gemfile

```diff
@@ -16,7 +16,7 @@ gem 'figaro'
 gem 'config'
 gem 'pg'
 gem 'json-schema'
-gem 'active_model_serializers'
+gem 'active_model_serializers', '0.10.4' # breaking changes in 0.10.5 and 0.10.6 -- 8/3/17
```

Comment:
A bunch of specs broken probably because of: https://github.com/rails-api/active_model_serializers/blob/0-10-stable/CHANGELOG.md